scripts/pi-hole/php/FTL.php

```diff
@@ -6,88 +6,93 @@
 *  This file is copyright under the latest version of the EUPL.
 *  Please see LICENSE file for your rights under this license. */
 
-$piholeFTLConfFile = "/etc/pihole/pihole-FTL.conf";
-
-function piholeFTLConfig()
-{
-	static $piholeFTLConfig;
-	global $piholeFTLConfFile;
-
-	if(isset($piholeFTLConfig))
-	{
-		return $piholeFTLConfig;
-	}
-
-	if(is_readable($piholeFTLConfFile))
-	{
-		$piholeFTLConfig = parse_ini_file($piholeFTLConfFile);
-	}
-	else
-	{
-		$piholeFTLConfig = array();
-	}
-
-	return $piholeFTLConfig;
+const DEFAULT_FTLCONFFILE = "/etc/pihole/pihole-FTL.conf";
+const FTL_API_IP = "127.0.0.1";
+const FTL_API_PORT = 4711;
```

Comment:
This should not be hard-coded (although it was like this before):
https://docs.pi-hole.net/ftldns/configfile/#ftlport

Reply:
I see.. this was meant as a `DEFAULT` value and could be overwritten latter by parsing `portfileContents`

Reply:
Exactly.

It was hard coded before, but inside a variable.

Now, there's a constant holding the **default** value.

The new function uses whatever you want. The default value will be used only if no values were passed.

Reply:
Done.